docs/introduction/store.md

```diff
@@ -0,0 +1,28 @@
+The `store` is comprised of three parts and is your main entry point for all things Rodux. It is created via [`Store.new`](../api-reference.md#storenew).
+
+### State
+This is the object tree that contains all of the data backing the business logic of your application. It is represented as a Lua table and it can be read via [`Store:getState`](../api-reference.md#storegetstate).
```

Comment:
In general I think we should stay away from calling the state an 'object tree'. It's totally possible to have your top-level reducer not be a tree!